src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx

```diff
@@ -1,21 +1,35 @@
-import _ from 'lodash';
+import React from 'react';
 import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
 import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
-export const SectionsWheelPickerDriver = (props: ComponentProps) => {
-  const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
-  const sectionsDrivers = _.map(sections, (_, index) => {
-    const sectionTestID = `${props.testID}.${index}`;
-    return WheelPickerDriver({
-      renderTree: props.renderTree,
-      testID: sectionTestID
-    });
-  });
 
-  
+type CustomSectionsTestIds = {
+  customSectionTestIds?: string[]
+};
+
+export const SectionsWheelPickerDriver = (props: ComponentProps & CustomSectionsTestIds) => {
+  const {customSectionTestIds, ...others} = props;
+  const driver = useComponentDriver<SectionsWheelPickerProps>(others);
 
+  let sectionsDrivers: (ReturnType<typeof WheelPickerDriver>)[];
+  const {renderTree} = props;
+  if (customSectionTestIds) {
+    sectionsDrivers = customSectionTestIds.map(testID => {
+      return WheelPickerDriver({
+        renderTree,
+        testID
+      });
+    });
+  } else {
+    // Default sections test ids.
+    sectionsDrivers = React.Children.toArray(driver.getElement().props.children).map((_section, index) => {
```

Comment:
Why all of this is necessary if you don't check if the child is of type WheelPicker? What's the difference with iterating over `driver.getProps().children as SectionsWheelPickerProps`?

Reply:
Regarding the props. The props that return from getProps are not necessarily SectionsWheelPickerProps they're just the props of the element. (We will probably do something with this function later in the future so I got the props in a different way).
On second look I thinks there might still be a problem here because I'm essentially assuming all the children are WheelPickerSections anyway but thats like the default behavior.
I could change it to always take the testID from the children and hope its a section and not something else we if we ever add anything else there.